### ISSUE TITLE
Disabled RadioGroup.Option should have attribute aria-disabled=true

### ISSUE THREAD

**Issue by zoltangy** [tailwindlabs/headlessui#515]
### What package within Headless UI are you using?

@headlessui/react

### What version of that package are you using?

v1.2.0

### What browser are you using?

Chrome

### Reproduction repository

-

### Describe your issue

A disabled RadioGroup.Option should have attribute aria-disabled=true.
Also, when disabling the whole RadioGroup, it should have the attribute aria-disabled=true.

**Comment by RobinMalfait** [comment]
Hey! Thank you for your feature suggestion!
Much appreciated! 🙏

This should be fixed, and will be available in the next release.
You can already try it using `npm install @headlessui/react@dev` or `yarn add @headlessui/react@dev`.
